microhapulator/data/template.html

```diff
@@ -115,15 +115,11 @@ <h2>Read QA/QC</h2>
             {% if read_length_table is none %}
                 {% if "r1readlen" in plots %}
                     <p>The following histograms show the distribution of R1 and R2 read lengths for each sample.</p>
-                    {% for r1plot, r2plot in zip(plots["r1readlen"], plots["r2readlen"]) %}
-                    <img src={{r1plot}} class="small" />
-                    <img src={{r2plot}} class="small" />
-                    {% endfor %}
+                    <img src="analysis/r1-read-lengths.png"/>
+                    <img src="analysis/r2-read-lengths.png"/>
```

Comment:
No need to use a for loop here since there is only one plot for r1 and one plot for r2